core-libs/setup/ssr/logger/transformers/utils/transformers-utils.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { TransformerConfig } from '../default-transformers';
+import { ExpressLogTransformer } from '../express-log-transformer';
+
+export interface ExpressLogTransformerConfig {
+  replace?: TransformerConfig;
+  remove?: [keyof TransformerConfig];
+}
+
+export function getLogTransformers(
+  transformers: ExpressLogTransformerConfig | undefined,
+  defaultTransformers: TransformerConfig
+): ExpressLogTransformer[] {
+  let parsedTransformers: TransformerConfig = {
+    ...defaultTransformers,
+    ...transformers?.replace,
+  };
+  if (transformers) {
+    transformers.remove?.forEach((transformerToBeRemoved) => {
+      delete parsedTransformers[transformerToBeRemoved];
+    });
+  }
+  return parseTransformers(parsedTransformers);
+}
+
+export function parseTransformers(
```

Comment:
nitpick: we don't "parse" those transformers, I believe 🤔 naming issue IMHO, but don't understand the author's intention, so hard to propose alternative

Reply:
Yeah, naming is the hardest part of the programming 😅 I didn't find a better name and didn't want to spend any more time on this PoC. We can go through this together and discuss it.